api/v1beta1/zz_generated.deepcopy.go

```diff
@@ -22,7 +22,7 @@ limitations under the License.
 package v1beta1
 
 import (
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
Not sure why this is changing since it'd still be named `v1`. Maybe go autosave changed some imports?

Reply:
fixed it... I guess my editor was doing something.